cmd/snap-confine/seccomp-support.c

```diff
@@ -423,7 +435,12 @@ static int parse_line(char *line, struct seccomp_args *sargs)
 		if (errno != 0)
 			return PARSE_ERROR;
 
-		sargs->arg_cmp[sargs->length] = SCMP_CMP(pos, op, value);
+		if (op == SCMP_CMP_MASKED_EQ)
```

Comment:
Can you please explain the reasoning behind this? I'm just unfamiliar and I'd like to understand it better.